npm/metrics/prometheus_metrics.go

```diff
@@ -1,31 +1,31 @@
 package metrics
 
 import (
+	"fmt"
 	"net/http"
 
 	"github.com/Azure/azure-container-networking/log"
 	"github.com/prometheus/client_golang/prometheus"
 	"github.com/prometheus/client_golang/prometheus/promhttp"
+	dto "github.com/prometheus/client_model/go"
 )
 
 const namespace = "npm"
 
 // Prometheus Metrics
 // Gauge metrics have the methods Inc(), Dec(), and Set(float64)
-// Summary metrics has the method Observe(float64)
+// Summary metrics have the method Observe(float64)
 // For any Vector metric, you can call With(prometheus.Labels) before the above methods
 //   e.g. SomeGaugeVec.With(prometheus.Labels{label1: val1, label2: val2, ...).Dec()
 var (
-	NumPolicies            prometheus.Gauge
-	AddPolicyExecTime      prometheus.Summary
-	NumIPTableRules        prometheus.Gauge
-	AddIPTableRuleExecTime prometheus.Summary
-	NumIPSets              prometheus.Gauge
-	AddIPSetExecTime       prometheus.Summary
-	NumIPSetEntries        prometheus.Gauge
-
-	// IPSetInventory should not be referenced directly. Use the functions in ipset-inventory.go
-	IPSetInventory *prometheus.GaugeVec
+	numPolicies        prometheus.Gauge
+	addPolicyExecTime  prometheus.Summary
+	numACLRules        prometheus.Gauge
```

Comment:
We are currently generating some Alerts based on these metric values, we are checking for total number of Iptable rules if it is greater than 2000 and number of ipsets. If we update the names of metrics here, we will have to update the metrics too . Please take a note of that..

Reply:
Where exactly are these alerts?

Reply:
Nevermind, I see their reference in your work item